pinecone/client.go

```diff
@@ -132,6 +400,62 @@ func (c *Client) ListIndexes(ctx context.Context) ([]*Index, error) {
 	return indexes, nil
 }
 
+// CreatePodIndexRequest holds the parameters for creating a new pods-based index.
+//
+// Fields:
+//   - Name: The name of the pods-based index to be created.
+//   - Dimension: The dimension of the index (must match [dimensionality] of upserted vectors).
```

Comment:
The `[]` brackets link to the resources at the bottom of the doc comment right? That's pretty cool.

Reply:
No, actually! It hyperlinks the text. The bottom stuff doesn't show in the rendered Godoc documentation; just in the code.